smtp/client.js

```diff
@@ -68,66 +121,89 @@ class Client {
 		this.smtp.connect(connect);
 	}
 
+	/**
+	 * @param {Message|MessageStack} msg msg
+	 * @param {function(Error, MessageStack): void} callback callback
+	 * @returns {void}
+	 */
 	send(msg, callback) {
-		if (
-			!(msg instanceof message.Message) &&
-			msg.from &&
-			(msg.to || msg.cc || msg.bcc) &&
-			(msg.text !== undefined || this._containsInlinedHtml(msg.attachment))
-		) {
-			msg = message.create(msg);
+		/**
+		 * @param {MessageStack} m message stack
+		 * @returns {boolean} can make message
+		 */
+		const canMakeMessage = m => {
```

Comment:
instead of recreating the function everytime, what about pulling this out into a private method?

Reply:
done